.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  run:
+    name: BuildAndTest
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system: [ macos-latest, ubuntu-latest, windows-latest ]
+        ocaml-version: [ '4.09.0', '4.08.1', '4.06.1']
+    steps:
+    - uses: actions/setup-node@v1
+    - uses: giltho/setup-ocaml@master
+      with:
+        ocaml-version: ${{ matrix.ocaml-version }}
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - run: opam pin add lsp.dev -n .
+    - run: opam pin add ocaml-lsp-server.dev.dev -n .
```

Comment:
Why is `.dev` doubled here? Also, isn't `opam install . --deps-only` enough? Why the extra pin step?

Reply:
The double `.dev` is a typo, and to be honest I don't really remember the process there. I'll change it and see if anything fails

Reply:
Ah, the extra pin step comes from here:
https://github.com/marketplace/actions/setup-ocaml

Reply:
I changed the double .dev but not the pin dev, it follow's the action's documentation, I don't know if I should remove it or not

Reply:
I pushed the simplified form and it seems to work just fine.